README.md

```diff
@@ -1,33 +1,44 @@
 # Docker 101
 ## Introduction to containerization for data science
 
-This repo is part of the introductory workshop on [Docker][docker] focussing specifically on containerization for data-science applications. This workshop is offered under the [Data Science for Albertans program](dsfa) at [Cybera][cybera]. It contains the setup/config files to build and successfully run a [JupyterLab][jupyterlab] container using [Docker][docker]. [JupyterLab][jupyterlab] provides a web-based interactive development environment for jupyter notebooks, code, and data. It’s widely used in different data-science applications. To learn more about JupyterLab, visit the link [here](jupyterlab). 
+This repo is part of the introductory workshop on [Docker][docker] focusing specifically on containerization for data-science applications. This workshop is offered as a part of the [Applied Data Science Lab program](adsl) at [Cybera][cybera]. It contains the setup/config files to build and successfully run a [JupyterLab][jupyterlab] container using [Docker][docker]. [JupyterLab][jupyterlab] provides a web-based interactive development environment for jupyter notebooks, code, and data. It’s widely used to prototype and develop different data-science applications. To learn more about JupyterLab, visit the link [here](jupyterlab). 
 
 ## Table of Contents
-1. [Introduction to Docker](#1)
-2. [Docker Installation](#2)
+1. [Introduction](#1)
+2. [Installation](#2)
 3. [Need for Containerization](#3)
 4. [Benefits for Data Science Enthusiasts](#4)
-5. [Setup](#5)
+5. [Build Jupyter Lab Container](#5)
 
 <a id="1"></a>
-### 1. Introduction to Docker
-[Docker](docker) is a open platform for developing, shipping and running applications. It reduces delay between writing code and running it in production.
-For starters, the [official Docker guide][guide] provides a good head start to using [Docker][docker]. 
+## 1. Introduction
+<a id="1.1"></a>
+### 1.1. Docker
+[Docker](docker) is an open-source platform for developing, shipping, and running applications. It reduces the delay between writing code and deploying it in production.
+For starters, the [official Docker guide][guide] provides a good head start on using `Docker`. 
+<a id="1.2"></a>
+### 1.2. Docker Compose
+[Docker Compose](compose) is a tool that enables running multiple `Docker` containers. `Docker Compose` is defined as a YAML file to configure different applications' services.  
 
 <a id="2"></a>
-### 2. Docker Installation 
-- Download installation wizard from [Dockerhub][install-docker]. 
+## 2. Installation
+### 2.1. Docker 
```

Comment:
I'm not sure but if working on the RAC or on an ubuntu system of any kind I usually just install the docker engine and compose since installing docker desktop is more for mac/pc to ensure compatibility. Since you work in the cmd line only with RAC it may be easier to follow:
https://docs.docker.com/engine/install/ubuntu/
... this one also installs docker compose as a plugin in the same step as installing the docker engine.